conf/templates/rzg2l-sbc/local.conf.sample

```diff
@@ -305,3 +305,6 @@ PREFERRED_RPROVIDER_wireless-regdb-static = "wireless-regdb"
 
 # Accepting commercial license flags for gstreamer build
 LICENSE_FLAGS_ACCEPTED:append = " commercial"
+
+# Disable generating JSON SPDX for images
+INHERIT:remove = "create-spdx"
```

Comment:
Add a function to check if `create-spdx` is in INHERIT variable, if yes then deploy SPDX artifacts to the host/build.